Tools/LyTestTools/ly_test_tools/_internal/pytest_plugin/exceptions.py

```diff
@@ -0,0 +1,18 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Exceptions that can occur while interacting with a Launcher
+"""
+
+
+class LyTestToolsException(Exception):
+    """ Indicates that an Exception resulted from a LyTT Test """
+
+    def __init__(self, inner_exception=None):
+        self.inner_exception = inner_exception
+
+    def __str__(self):
+        return f"This exception was generated by a LyTT test. The inner exception is {self.inner_exception}"
```

Comment:
same wording.  Also just wondering how the formatting looks with the inner exception